### PR TITLE
ADBDEV-6635: Align LIKE INCLUDE STORAGE docs with existing implementation

### PR DESCRIPTION
**Pull request by silent-observer**
Align LIKE INCLUDE STORAGE docs with existing implementation

Current documentation related to LIKE INCLUDE STORAGE is confusing and
incomplete. This patch adds details on which exact settings are copied if
INCLUDE STORAGE is specified.

### CODE REVIEW

**Comment by BenderArenadata** [comment]
Allure report https://allure.adsw.io/launch/84668

File: doc/src/sgml/ref/create_table.sgml [hunk 1]
```diff
@@ -443,7 +443,10 @@ Where column_reference_storage_directive is:
       default behavior is to exclude <literal>STORAGE</> settings, resulting
       in the copied columns in the new table having type-specific default
       settings.  For more on <literal>STORAGE</> settings, see
-      <xref linkend="storage-toast">.
+      <xref linkend="storage-toast">. <literal>ENCODING</> settings and
+      table-wide append-optimized options (such as such as
```
Comment (red1452) [inline doc/src/sgml/ref/create_table.sgml]: such as such as

File: gpdb-doc/dita/ref_guide/sql_commands/CREATE_TABLE.xml [hunk 2]
```diff
@@ -343,8 +343,8 @@ CREATE [ [GLOBAL | LOCAL] {TEMPORARY | TEMP} | UNLOGGED ] TABLE [IF NOT EXISTS]
             distribution policy. Unlike <codeph>INHERITS</codeph>, the new table and original table
             are completely decoupled after creation is complete.</pd>
           <pd>
-            <note>Storage properties like append-optimized or partition structure are not
-              copied.</note>
+            <note>Partition structure is not copied. Storage properties like append-optimized are
+            only copied if <codeph>INCLUDING STORAGE</codeph> is specified.</note>
```
Comment (red1452) [inline gpdb-doc/dita/ref_guide/sql_commands/CREATE_TABLE.xml]: I would add information that leaf partitions inherit storage settings from the root partition.

**Comment by BenderArenadata** [comment]
Allure report https://allure.adsw.io/launch/84811

File: gpdb-doc/dita/ref_guide/sql_commands/CREATE_TABLE.xml (hunk 2 above)
Comment (silent-observer) [reply gpdb-doc/dita/ref_guide/sql_commands/CREATE_TABLE.xml]: Done

**Comment by BenderArenadata** [comment]
Allure report https://allure.adsw.io/launch/84814

**Review by red1452 — DISMISSED** [review]

File: doc/src/sgml/ref/create_table.sgml (hunk 1 above)
Comment (bimboterminator1) [inline doc/src/sgml/ref/create_table.sgml]: How newly added sentence is linked to previous information. Seems unnatural

Comment (silent-observer) [reply doc/src/sgml/ref/create_table.sgml]: Fixed

**Review by bimboterminator1 — APPROVED** [review]

**Comment by BenderArenadata** [comment]
Allure report https://allure.adsw.io/launch/85446

**Review by red1452 — APPROVED** [review]

**Comment by BenderArenadata** [comment]
Allure report https://allure.adsw.io/launch/85494

**Pull request merged**